### PR TITLE
Fix go 1.18 installation compatibility issue

### PR DESCRIPTION
**Pull request by wilsonwu**
Fix go 1.18 installation compatibility issue:
image

solve #33

### CODE REVIEW

**Comment by wilsonwu** [comment]
@lyonlai please help to review and merge, thank you very much!

**Comment by lyonlai** [comment]
thanks for the contribution @wilsonwu . I'll take a look soon

**Review by lyonlai — CHANGES_REQUESTED** [review]
changes LGTM but the build's failing. will require some extra effort from you. @wilsonwu

**Comment by wilsonwu** [comment]
@lyonlai seems the job broken for long time, I tried to solve it, please help to run it.

**Comment by wilsonwu** [comment]
@lyonlai please help to rerun, already fixed in my repo:
image

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -67,13 +67,13 @@ jobs:
...
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
-          go-version: 1.16
+          go-version: 1.17
```
Comment (lyonlai) [inline .github/workflows/main.yml]: might as well change this to 1.18?

**Review by lyonlai — CHANGES_REQUESTED** [review]
missing the go-releaser for 1.18. others LGTM

File: .github/workflows/main.yml (hunk 1 above)
Comment (wilsonwu) [reply .github/workflows/main.yml]: make sense

Comment (wilsonwu) [reply .github/workflows/main.yml]: Anything else I need to do for go-releaser? Sorry for that I have no experience on such cases.

**Review by lyonlai — APPROVED** [review]

**Comment by lyonlai** [comment]
Thanks you for your. contribution @wilsonwu

**Pull request merged**